src/ClientResolver.php

```diff
@@ -412,6 +422,21 @@ public static function _apply_debug($value, array &$args, HandlerList $list)
         }
     }
 
+    public static function _apply_stats($value, array &$args, HandlerList $list)
+    {
+        $defaults = array_fill_keys(
```

Comment:
These are still defaulting to all on. If I specify `['timer' => true]`, then you'd also add in retries and http as true unless I explicitly set them to `false`. I think the less surprising thing to do here would be to require users to explicitly opt-in to stats features when they provide an associative array.